### DISCUSSION TITLE
Add Branch Coverage metric

### DISCUSSION THREAD

**Discussion by Will-GuiSKJ** [FortuneN/FineCodeCoverage#120] [Ideas]
First off, thank you so much for this Extension!

Would it be possible for you to add Branch coverage metrics alongside LoC coverage? I think both of these metrics together are crucial to really understand how robust our unit tests are.

**Comment by tonyhallett** [comment]
See issue #29. It will be added soon

**Reply by Will-GuiSKJ** [reply]
Thanks!
